modules/apps/users-admin/users-admin-test/src/testIntegration/java/com/liferay/users/admin/search/test/UserReindexerPerformanceOfLargeUserGroupInManySitesTest.java

```diff
@@ -284,7 +285,9 @@ protected void measure(
 
```

Comment:
corrected for this: https://github.com/liferay-core-infra/liferay-portal/pull/6781#discussion_r1536018269